compiler-core/src/format/tests.rs

```diff
@@ -3157,6 +3157,17 @@ fn expr_panic_as() {
     );
 }
 
+#[test]
+fn expr_panic_as_value() {
+    assert_format!(
+        r#"fn main() {
+  let x = "panicking" <> "with a value"
+  panic as x
+}
+"#
+    );
+}
```

Comment:
Could we have Erlang and JS tests for `todo as`, and tests for the formatter too please? Thank you

Reply:
Sure. Will add more tests.